frontend/src/pages/RunDetails.test.tsx

```diff
@@ -56,7 +60,10 @@ describe('RunDetails', () => {
   }
 
   beforeAll(() => jest.spyOn(console, 'error').mockImplementation());
-  afterAll(() => jest.resetAllMocks());
+  afterAll(() => {
+    jest.resetAllMocks();
```

Comment:
This is a good, but shouldn't it be `afterEach` ?

In that case, you could remove all of the `mockClear` and `mockReset`s from the `beforeEach` below.

Reply:
Yes it should, fixed.
Now using `jest.clearAllMocks` before each test, and `jest.resetAllMocks` after each.

Reply:
I think that might even be more than is necessary. `reset` should also clear, so you'd only be getting one extra clear called before the first test, and that likely doesn't get you anything